client/src/popups/showAnalyticsAllowPopup.ts

```diff
@@ -44,4 +38,6 @@ export async function showAnalyticsAllowPopup({
   await config.update("telemetry", isAccepted, ConfigurationTarget.Global);
 
   await context.globalState.update(PREVIOUSLY_SHOWN_TELEMETRY_LABEL, true);
+
+  client?.sendNotification("custom/telemetryConsent", isAccepted);
```

Comment:
question: I wonder if this change will record the consent status for people who already gave (or rejected) telemetry before? would we collect this data for older users?